tpgtools/types/type.go

```diff
@@ -86,7 +76,7 @@ func (t Type) String() string {
 	case "":
 		return "<nil>"
 	default:
-		return fmt.Sprintf("undefined type: %s", t.typ)
+		return fmt.Sprintf("undefined type: %T", t.typ)
```

Comment:
Won't this print the Go type, eg `*openapi.Schema` when we want the actual value contained within? It's possible that `typ` used to be a string or something, since I'm not sure this worked when it was `%s`.

Reply:
golang was complaining to be about %s, switching to %T seemed to fix that. Maybe I can add a testcase to debug